eslint.config.js

```diff
@@ -25,6 +25,14 @@ export default tseslint.config(
         { allowConstantExport: true },
       ],
     },
+    settings: {
+      'import/resolver': {
+        alias: {
+          map: [['@shared', './src/shared']],
```

Comment:
**Question:** Does it worth having different entries for `@component` and `@theme`?

Reply:
looks good, lets go like this